src/seq/nucleotide.jl

```diff
@@ -679,6 +679,7 @@ end
 # Construct a subesequence
 getindex{T}(seq::NucleotideSequence{T}, r::UnitRange) = NucleotideSequence{T}(seq, r)
 
+sub{T}(seq::NucleotideSequence{T}, r::UnitRange) = NucleotideSequence{T}(seq, r)
```

Comment:
Thanks for reminding me! AA sequences sometimes slip my mind because I tend to work with DNA.